app/controllers/api/v1/votes_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Api::V1::VotesController < ApiController
+	before_action :authenticate_user
+
+  def create
+		existing_vote = Vote.where(user_id: current_user.id, review_id: params['review_id']).first
```

Comment:
just use `find_by`. `.where` with `.first` is less ideal

Reply:
Changed .where to .find_by